### PR TITLE
wdio-timeline-reporter: Added timeline reporter as a SUPPORTED_REPORTER

### PR DESCRIPTION
**Pull request by benzaremean**
## Proposed changes

[//]: # (Added wdio-timeline-reporter as a supported reporter)

## Types of changes

[//]: # (Gives ability to install new wdio-timeline-reporter using CLI)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ x] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-cli/src/config.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ export const SUPPORTED_REPORTER = [
     ' json - https://www.npmjs.com/package/wdio-json-reporter',
     ' concise - https://www.npmjs.com/package/wdio-concise-reporter',
     ' testrail - https://www.npmjs.com/package/wdio-testrail-reporter',
-    ' mochawesome - https://www.npmjs.com/package/wdio-mochawesome-reporter'
+    ' mochawesome - https://www.npmjs.com/package/wdio-mochawesome-reporter',
+    ' timeline - https://www.npmjs.com/package/wdio-mochawesome-reporter'
```
Comment (christian-bromann) [inline packages/wdio-cli/src/config.js]: should be
```
' timeline - https://www.npmjs.com/package/wdio-timeline-reporter'
```

Comment (benzaremean) [reply packages/wdio-cli/src/config.js]: 🤦‍♂️ had to delete previous commit and re-committed as my git user was not set up. Will make change now

**Comment by emilyrohrbough** [comment]
Note: the wdio-timeline-reporter is a utility module. It offers the timeline reporter and a screenshot tool. It should be noted that using this package is 'heavier' than other reporters.

**Comment by christian-bromann** [comment]
> It should be noted that using this package is 'heavier' than other reporters.

Please note this in the readme of that project. Thanks!

**Pull request merged**